modules/users/controller.js

```diff
@@ -81,11 +94,41 @@ export const current = async (req, res, next) => {
 		}
 		return res
 			.status(200)
-			.json({ user: user.email, subscription: user.subscription });
+			.json({ user: user, subscription: user.subscription });
```

Comment:
jesli przekazujesz calego usera, to nie ma sensu dodatkowo przekazywac subscription 😉